components/messages.json

```diff
@@ -11,10 +11,41 @@
         "text": "@theme-text-secondary-normal"
       }
     },
+    "bodyText-hyperlink": {
```

Comment:
can we add the Figma link to these component.
```
"message":{
     "figma": "http...",
     "comment": "Color tokens for..."
     "bodyText-primary": ....
     ....
}
```

Reply:
and... is "Color tokens for space list" the right comment for this component? or is it for the "Color tokens for message blocks inside a space conversation"